docs/code/os-dev.md

```diff
@@ -124,12 +124,18 @@ The first argument is the message which is formatted like `printf`. This means t
 
 ### Retrieving logs {#retrieving-logs}
 
-By default these messages are not shown. To see them you need to set the `G_MESSAGES_DEBUG` environment variable to the log domain you're interested in. Usually you'll set it to `all` to log everything. [More info](https://developer.gnome.org/glib/stable/glib-running.html).
+By default debug messages are not shown. To see them you need to set the `G_MESSAGES_DEBUG` environment variable to the log domain you're interested in.
+[More info on environmental variables](https://www.digitalocean.com/community/tutorials/how-to-read-and-set-environmental-and-shell-variables-on-a-linux-vps)
+Usually you'll set it to `all` to log everything. [More info on Running and debugging GLib Applications](https://developer.gnome.org/glib/stable/glib-running.html).
 
-Example:
+Run your application with debugging enabled:
 ```
 G_MESSAGES_DEBUG=all ./Application
```

Comment:
This is not very clear, it would only work if typed literally if the user had an executable named "Application" in the current directory.  When compiling from source the executable might be in `<`*`source`*`>/build/` or `<`*`source`*`>/build/src`.  

Indication of the need to substitute something is not consistent in this document nor the way of indicating what to substitute.  `your-app`, `appname` (for a package), `Application` and `<packagename>` has been used.  I suggest using brackets and italics with consistent placeholders.

Reply:
Good point! I've now used `<app>`. 
After this merge I'll do a separate PR to make the other substitution points more consistent.